app/index.js

```diff
@@ -1,22 +1,26 @@
-import React, { View, Text } from 'react-native';
+import React, { View, Text,
+
+  TextInput, Image
```

Comment:
**<a href='http://eslint.org/docs/rules/no-unused-vars'>no-unused-vars</a>**: "TextInput" is defined but never used